src/emulator/extensionsEmulator.ts

```diff
@@ -324,7 +324,7 @@ export class ExtensionsEmulator implements EmulatorInstance {
         );
         table.push([
           apiToWarn.apiName,
-          apiToWarn.instanceIds,
+          apiToWarn.instanceIds.toString(),
```

Comment:
```suggestion
          apiToWarn.instanceIds.join(", "),
```
[].toString doesn't put spaces between elements, so its kind of ugly